ksql-engine/src/test/java/io/confluent/ksql/analyzer/AggregateAnalyzerTest.java

```diff
@@ -14,205 +14,267 @@
 
 package io.confluent.ksql.analyzer;
 
-import static io.confluent.ksql.testutils.AnalysisTestUtil.analyzeQuery;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.mock;
 
+import com.google.common.collect.ImmutableList;
 import io.confluent.ksql.function.InternalFunctionRegistry;
-import io.confluent.ksql.metastore.MetaStore;
-import io.confluent.ksql.parser.tree.ComparisonExpression;
+import io.confluent.ksql.parser.tree.DereferenceExpression;
 import io.confluent.ksql.parser.tree.Expression;
-import io.confluent.ksql.parser.tree.ExpressionTreeRewriter;
-import io.confluent.ksql.util.AggregateExpressionRewriter;
-import io.confluent.ksql.util.MetaStoreFixture;
-import org.junit.Assert;
+import io.confluent.ksql.parser.tree.FunctionCall;
+import io.confluent.ksql.parser.tree.Literal;
```

Comment:
nit: unused imports Literal and StringLiteral

Reply:
Not like me ... arh, IntelliJ has turned of my check-in checks.